files/es/mdn/writing_guidelines/page_structures/page_types/http_header_page_template/index.md

```diff
@@ -0,0 +1,180 @@
+---
+title: Plantilla de página de encabezado HTTP
+slug: MDN/Writing_guidelines/Page_structures/Page_types/HTTP_header_page_template
+---
+
+{{MDNSidebar}}
+
+> **Nota:** _Elimina toda esta nota explicativa antes de publicar_
+>
+> ---
+>
+> **Metadatos de la página:**
+>
+> Los metadatos en la parte superior de la página se utilizan para definir "metadatos de la página".
+> Los valores deben actualizarse adecuadamente para el elemento en particular.
+>
+> ```md
+> ---
+> title: <NombreDelElemento>
+> slug: Web/SVG/Element/NombreDelElemento
+> page-type: svg-element
+> status:
+>   - experimental
+>   - deprecated
+>   - non-standard
+> browser-compat: svg.elements.NombreDelElemento
```

Comment:
Maybe this is from the last template translation

```suggestion
> title: NombreDeLaCabecera
> slug: Web/HTTP/Headers/NameOfTheHeader
> page-type: http-header
> status:
>   - experimental
>   - deprecated
>   - non-standard
> browser-compat: path.to.feature.NameOfTheHeader
```

Reply:
Fue un error al copiarlo de otra traducción